sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/_tests_/disbursement-receipts-integration.scheduler.e2e-spec.ts

```diff
@@ -717,6 +740,95 @@ describe(
+
+      // Assert
+      expect(result).toStrictEqual([
+        "Completed disbursement receipts integration.",
+      ]);
+      const downloadedFile = path.join(
+        process.env.ESDC_RESPONSE_FOLDER,
+        NON_MATCHING_RECORDS_FILE,
+      );
+      expect(
+        mockedJob.containLogMessages([
+          `Processing file ${downloadedFile}.`,
+          `Document number ${SHARED_DOCUMENT_NUMBER} at line 2 not found in SIMS.`,
+          `Document number ${SHARED_DOCUMENT_NUMBER} at line 3 not found in SIMS.`,
+          `Document number ${SHARED_DOCUMENT_NUMBER} at line 4 not found in SIMS.`,
+          `Processing file ${downloadedFile} completed.`,
+          `Processing provincial daily disbursement CSV file on ${FILE_DATE}.`,
+        ]),
+      ).toBe(true);
+      // Assert imported receipts.
+      const { feReceipt, bcReceipt } = await getReceiptsForAssert(
+        FILE_DATE,
+        SEQUENCE_NUMBER,
+      );
+      // BC receipt should not be present.
+      expect(bcReceipt).not.toBeDefined();
+      // BC receipt should not be present.
+      expect(feReceipt).not.toBeDefined();
+      // Notification record.
+      const createdNotification = await db.notification.findOne({
+        select: {
+          id: true,
+          dateSent: true,
+          messagePayload: true,
+          notificationMessage: { id: true, templateId: true },
+        },
+        relations: { notificationMessage: true },
+        where: {
+          dateSent: IsNull(),
+          notificationMessage: {
+            id: NotificationMessageType.MinistryNotificationProvincialDailyDisbursementReceipt,
+          },
+        },
+      });
+      expect(createdNotification.messagePayload).toStrictEqual({
+        template_id: createdNotification.notificationMessage.templateId,
+        email_address: TEST_EMAIL,
+        personalisation: {
+          application_file: {
+            file:
+              "RnVsbCBUaW1lIEJDIFN0dWRlbnQgTG9hbixGdWxsIFRpbWUgQkMgU3R1ZGVudCBHcmFudCxGdWxsIFRpbWUgQkMgVG90YWwsUGFydCBUaW1lIEJDIFN0dWRlbnQgR3JhbnQsUGFydCBUaW1lIEJDIFRvd" +
+              "GFsLEJDIFRvdGFsLFRvdGFsIFJlY29yZHMsRmlsZSBEYXRlLEJhdGNoIFJ1biBEYXRlLFNlcXVlbmNlIE51bWJlcg0KMCwwLDAsMCwwLDAsMCwyMDI0LTAxLTMxLCwzMjI4",
+            filename: `Daily_Disbursement_File_${FILE_DATE}_${SEQUENCE_NUMBER}.csv`,
+            sending_method: "attach",
+          },
+        },
+      });
+      // Verify the file content as expected.
+      const file =
+        createdNotification.messagePayload["personalisation"][
+          "application_file"
+        ]["file"];
+      const fileContent = Buffer.from(file, "base64").toString("ascii");
+      expect(fileContent).toContain(
+        "Full Time BC Student Loan,Full Time BC Student Grant,Full Time BC Total,Part Time BC Student Grant,Part Time BC Total,BC Total,Total Records,File Date,Batch Run Date,Sequence Number",
+      );
+      expect(fileContent).toContain("0,0,0,0,0,0,0,2024-01-31,,3228");
```

Comment:
The date and sequence number can be used a constant so that it can be used across many tests in the file.

Reply:
Please use the constant that is already defined.

Reply:
I believe it is from existing code.